<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>线路导航系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>561</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_1"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>查找地点</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="graphview" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>起始地:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>君子兰桥</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>目的地:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>41</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>出行方式:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>出行方案:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>140</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>前往目的地</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>586</x>
      <y>189</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>美食</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>前往距离:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>用户评价:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>80</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>人均消费:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>超市</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>前往距离:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>用户评价:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>正在营业</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>酒店 </string>
     </attribute>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>用户评价:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>住店价格(天):</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_9">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_10">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>前往距离:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>370</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>花费时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>510</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>路费消耗:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>510</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>经过红绿灯个数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>510</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑地图</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>关于开发者</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action"/>
   <addaction name="action_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>编辑地点</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>编辑道路</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>关于开发者</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_1</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboBox_3</tabstop>
  <tabstop>comboBox_4</tabstop>
  <tabstop>comboBox_5</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>comboBox_6</tabstop>
  <tabstop>comboBox_7</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>comboBox_8</tabstop>
  <tabstop>comboBox_9</tabstop>
  <tabstop>comboBox_10</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
